src/containers/chat/mail-form/mail-form.js

```diff
@@ -29,9 +29,11 @@ export const MailForm = ({
         <span className={style.contactsTitle}>
           {CHAT[language].ourContacts}
         </span>
-        <span className={style.phoneNumbers}>
-          +{contacts.map((contact) => contact.phoneNumber)}
-        </span>
+        {contacts.map((contact, index) => (
+          <span className={style.phoneNumbers} key={index}>
```

Comment:
Please, don't use `index` as a key, use some another unique stable value from the `contact` object

Reply:
fixed